version: '3.4'

services:
  pythonfastapi:
    image: pythonfastapi
    build:
      context: ./PythonFastApi/
      dockerfile: ./PythonFastApi/Dockerfile
    ports:
      - 8811:8811
